### PR TITLE
Summary virtual

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

**Comment by cquiroz** [comment]
Fixes [sc-1760]

**Comment by cquiroz** [comment]
This includes a fix we'd need for testing. I'll just merge it as it is fairly small

File: explore/src/main/scala/explore/targets/TargetSummaryTable.scala [hunk 1]
```diff
@@ -241,11 +245,13 @@ object TargetSummaryTable extends TableHooks:
...
             )
           ),
-          PrimeTable(
+          PrimeVirtualizedTable(
             table,
+            _ => 32.toPx,
             striped = true,
             compact = Compact.Very,
             tableMod = ExploreStyles.ExploreTable |+| ExploreStyles.ExploreSelectableTable,
+            containerMod = ExploreStyles.ExploreFullHeightTable,
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/targets/TargetSummaryTable.scala:254]: Did you try `AutoHeightPrimeVirtualizedTable`? Maybe this class is not necessary.

Comment (cquiroz) [reply explore/src/main/scala/explore/targets/TargetSummaryTable.scala:254]: Ah I could try that, I thought it was auto row height

Comment (cquiroz) [reply explore/src/main/scala/explore/targets/TargetSummaryTable.scala]: Yeah that works, thanks

**Pull request merged**